components/match2/wikis/halo/match_group_input_custom.lua

```diff
@@ -177,6 +177,8 @@ end
 function MapFunctions.getExtraData(map, opponentCount)
 	return {
 		comment = map.comment,
+		points1 = map.points1,
```

Comment:
would we want a check that ensures points are only entered if score is set?

Reply:
I think there's might be some cases on the Halo wiki where only the points technically were added and not the score (Due to whatever information was available), so might be good to still support it if only that is known?

Reply:
hmm the display in this PR doesn't support that
so that would need to be adjusted too

Reply:
What Kano means is that points were added in the score parameters, the new parameters are in fact new and have not been used so far

Reply:
> would we want a check that ensures points are only entered if score is set?

As in, error in case they aren't? Not sure, since they won't be displayed without the score i think it should be fine.

Reply:
yeah the question was if we would want to let it error
just a question if it is wanted or not^^